modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -119,6 +128,18 @@ sshKey: |
 <7> Set the path to the installation disk drive, for example, `/dev/sda`.
 <8> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
 <9> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+<10> Activate a predefined set of cluster capabilities.
+<11> Add the additional cluster capabilities you want to enable here.
+
+[NOTE]
+====
+The following sets of cluster capabilities are predefined:
+
+`None`:: All capabilities are disabled.
```

Comment:
The godocs here [say][1] it:

> ... is an empty set enabling no optional capabilities

That's slightly different from disabling all capabilities, because:

```yaml
baselineCapabilitySet: None
additionalEnabledCapabilities:
- Insights
```

has `Insights` enabled, not disabled, etc.  I'd recommend just echoing the godocs here.

[1]: https://github.com/openshift/api/blob/0ee1471bcabbfefade29abeae5aab53366d0493f/config/v1/types_cluster_version.go#L289-L291

Reply:
ok